<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_lobby</class>
 <widget class="QWidget" name="client_lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lobbyBackground">
   <property name="geometry">
    <rect>
     <x>-180</x>
     <y>0</y>
     <width>1131</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="client_connector_resources.qrc">:/images/lobby_background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lobbyGamesImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="client_connector_resources.qrc">:/images/lobby_games.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitLobbyButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createGameButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinGameButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Game</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="gamesTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>451</width>
     <height>192</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>451</width>
     <height>192</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>451</width>
     <height>192</height>
    </size>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">Game</string>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">Creator</string>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">Players</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Welcome:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="playerName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="client_connector_resources.qrc"/>
 </resources>
 <connections/>
</ui>
